compiler+runtime/test/jank/cpp/try/pass-catch-cpp-char.jank

```diff
@@ -0,0 +1,13 @@
+;; Test catching C++ char exception
+(cpp/raw "void throw_char_z() { throw 'z'; }")
```

Comment:
These functions need to be wrapped in a namespace which is specific to the test. You're adding symbols into the global native namespace here, which could then conflict with other tests. This has led to issues in the past and it's not how we do it anymore. This should be:

```clojure
(cpp/raw "namespace jank::cpp::try_::pass_catch_cpp_char
          {
            void throw_char_z() { throw 'z'; }
          }")
```